packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -914,17 +922,27 @@ class RenderParagraph extends RenderBox
             assert(false);
           }
         }
-        newChildren.add(
-          SemanticsNode()
-            ..updateWith(config: configuration)
-            ..rect = currentRect,
-        );
+        final SemanticsNode newChild = (_cachedChildNodes?.isNotEmpty == true)
```

Comment:
Do you need the `== true`?

Reply:
Yeah, it is needed here. `_cachedChildNodes` may be null and then `_cachedChildNodes?.isNotEmpty` is also null, which is not a valid boolean expression.

Reply:
Oh of course, got it.